packages/generator-volto/README.md

```diff
@@ -64,6 +64,12 @@ You can provide an specific Volto version like:
 yo @plone/volto --volto=12.0.0-alpha.0
 ```
 
+You can also pass a Volto branch or tag:
+
+```bash
```

Comment:
We should use the generic name of a shell, as each flavor can have slightly different syntax highlighting.

````suggestion
```shell
````